src/coord/debug_command_names.h

```diff
@@ -11,5 +11,7 @@
  */
 static const char *coordCommandsNames[] = {
   "SHARD_CONNECTION_STATES",
+  "PAUSE_TOPOLOGY_UPDATER",
+  "RESUME_TOPOLOGY_UPDATER",
   NULL
```

Comment:
Consider using a single subcommand for both (with a third argument to distinguish)

Reply:
I prefer to implement it this way so I don't have to deal with invalid inputs